tests/unit/sagemaker/jumpstart/test_utils.py

```diff
@@ -1495,3 +1504,174 @@ def test_get_region_fallback_success(s3_bucket_name, s3_client, sagemaker_sessio
 def test_get_region_fallback_failure(s3_bucket_name, s3_client, sagemaker_session):
     with pytest.raises(ValueError):
         utils.get_region_fallback(s3_bucket_name, s3_client, sagemaker_session)
+
+
+class TestPresetConfigs:
+    @patch("sagemaker.jumpstart.accessors.JumpStartModelsAccessor.get_model_specs")
+    def test_get_jumpstart_presets_empty(
+        self,
+        patched_get_model_specs,
+    ):
+
+        patched_get_model_specs.side_effect = get_special_model_spec
+
+        assert utils.get_preset_names("mock-region", "gemma-model", "mock-model-version") == []
```

Comment:
would we not want to raise an exception here?

Reply:
I'm open to it either way, say a model doesn't have any configs in its metadata, we can raise an exception saying the model has empty config, or we treat empty list as the expected result.
For now I'll keep as is, let me know if you feel strongly we should raise an exception.